super.c

```diff
@@ -331,6 +635,5 @@ int simplefs_fill_super(struct super_block *sb, void *data, int silent)
     kfree(sbi);
 release:
     brelse(bh);
-
```

Comment:
Don't do this.

Reply:
I've made the following changes: when simplefs_parse_options fails, it will directly return the error code (ret), instead of using goto release

Reply:
You should keep the original blank line. Minimize the necessary changes.